Build error for graphene/LibOS/glibc-build/malloc/memusagestat

## Description of the problem

Tried to build Graphene on intel Core i7-6700K desktop with Ubuntu 18.04.5 LTS. 

- gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
- glibc (Ubuntu GLIBC 2.27-3ubuntu1.4) 2.27

Got the following build errors:

```
Building glibc, may take a while to finish. Warning messages may show up. If this process terminates with failures, see "glibc-build/build.log" for more information.
(CC=gcc CXX=g++ AS=gcc make -C glibc-build 2>&1 > build.log) && touch glibc-build/Build.success
//lib/x86_64-linux-gnu/librt.so.1: undefined reference to `__clock_settime@GLIBC_PRIVATE'
//lib/x86_64-linux-gnu/librt.so.1: undefined reference to `__clock_getcpuclockid@GLIBC_PRIVATE'
//lib/x86_64-linux-gnu/librt.so.1: undefined reference to `__clock_nanosleep@GLIBC_PRIVATE'
//lib/x86_64-linux-gnu/librt.so.1: undefined reference to `__clock_getres@GLIBC_PRIVATE'
collect2: error: ld returned 1 exit status
make[4]: *** [/home/dychen/temp/sgx/graphene/LibOS/glibc-build/malloc/memusagestat] Error 1
make[3]: *** [malloc/others] Error 2
make[2]: *** [all] Error 2
Makefile:68: recipe for target 'glibc-build/Build.success' failed
make[1]: *** [glibc-build/Build.success] Error 2
make[1]: Leaving directory '/home/dychen/temp/sgx/graphene/LibOS'
Makefile:13: recipe for target 'all' failed
make: *** [all] Error 2
```
The last few lines in the build.log shows:
```
gcc -Wl,-rpath-link=/home/dychen/temp/sgx/graphene/LibOS/glibc-build:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/math:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/elf:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/dlfcn:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/nss:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/nis:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/rt:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/resolv:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/mathvec:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/support:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/crypt:/home/dychen/temp/sgx/graphene/LibOS/glibc-build/nptl -pie -Wl,-O1 -nostdlib -nostartfiles -o /home/dychen/temp/sgx/graphene/LibOS/glibc-build/malloc/memusagestat    -Wl,-z,combreloc -Wl,-z,relro -Wl,--hash-style=both /home/dychen/temp/sgx/graphene/LibOS/glibc-build/csu/Scrt1.o /home/dychen/temp/sgx/graphene/LibOS/glibc-build/csu/crti.o `gcc  --print-file-name=crtbeginS.o` /home/dychen/temp/sgx/graphene/LibOS/glibc-build/malloc/memusagestat.o  -lgd -lpng -lz -lm -Wl,-dynamic-linker=/home/dychen/temp/sgx/graphene/LibOS/../Runtime/lib/ld-linux-x86-64.so.2  /home/dychen/temp/sgx/graphene/LibOS/glibc-build/libc.so.6 /home/dychen/temp/sgx/graphene/LibOS/glibc-build/libc_nonshared.a -Wl,--as-needed /home/dychen/temp/sgx/graphene/LibOS/glibc-build/elf/ld.so -Wl,--no-as-needed -lgcc  `gcc  --print-file-name=crtendS.o` /home/dychen/temp/sgx/graphene/LibOS/glibc-build/csu/crtn.o
../Rules:215: recipe for target '/home/dychen/temp/sgx/graphene/LibOS/glibc-build/malloc/memusagestat' failed
make[4]: Leaving directory '/home/dychen/temp/sgx/graphene/LibOS/glibc-2.31/malloc'
Makefile:470: recipe for target 'malloc/others' failed
make[3]: Leaving directory '/home/dychen/temp/sgx/graphene/LibOS/glibc-2.31'
Makefile:9: recipe for target 'all' failed
make[2]: Leaving directory '/home/dychen/temp/sgx/graphene/LibOS/glibc-build'
```

## Steps to reproduce

1. git clone https://github.com/oscarlab/graphene.git
commit d1e3f27060a06ea3654e315678922f75eb31a597 (HEAD -> master, origin/master, origin/HEAD)
2. sudo apt-get install -y build-essential autoconf gawk bison wget python3
3. cd graphene/
4. make

## Expected results

Build should succeed

## Actual results

Build failed with undefined references when linking memusagestat.

--- a/LibOS/Makefile
+++ b/LibOS/Makefile
@@ -77,6 +77,7 @@ $(BUILD_DIR)/Makefile: $(GLIBC_SRC)/.configured
 	CFLAGS=$$GLIBC_CFLAGS CPPFLAGS=$$GLIBC_CPPFLAGS \
 	../$(GLIBC_SRC)/configure --prefix=$(RUNTIME_DIR) \
 		--with-tls \
+		--without-gd \
 		--without-selinux \
 		--disable-test \
 		--disable-nscd \

